scripts/setup.sh

```diff
@@ -19,7 +19,7 @@ else
   exit 1
 fi
 echo "Generating DID..."
-npx ts-node scripts/gen-did/main.ts "${PAYMENT_ACCOUNT_SEED}"
+npx ts-node scripts/gen-did/main.ts "${PAYMENT_ACCOUNT_SEED}" || { echo 'my_command failed' ; exit 1; }
```

Comment:
'My command failed'?

Reply:
Maybe we should change this

Reply:
🤦🏻 I changed it but didn't commit. Fixed.